packages/astro/src/assets/fonts/utils.ts

```diff
@@ -46,11 +46,11 @@ export function renderFontSrc(sources: Exclude<unifont.FontFaceData['src'][numbe
 		.map((src) => {
 			if ('url' in src) {
 				let rendered = `url("${src.url}")`;
-				for (const key of ['format', 'tech'] as const) {
-					const value = src[key];
-					if (value) {
-						rendered += ` ${key}(${value})`;
-					}
+				if (src.format) {
+					rendered += ` format("${src.format}")`
```

Comment:
quotes wrapping wasn't strictly required though, i tested with chrome, firefox, and zen :)

Reply:
Good to know, I made this change based on the MDN docs which show format with quotes and tech without